### PR TITLE
Add Card component

### PR DESCRIPTION
**Pull request by ugogo**

### CODE REVIEW

File: packages/react/src/components/Card/index.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  <div
+    className={cx("Card", className, {
+      "is-selected": isSelected,
+      [`is-elevated--${elevation}`]: elevation,
+    })}
+    {...props}
+  />
+);
+
+Card.propTypes = {
```
Comment (obedparla) [inline packages/react/src/components/Card/index.js]: What about adding "Children" here?

File: packages/react/src/components/Card/index.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  isSelected: PropTypes.bool,
+};
+
+Card.defaultProps = {
+  className: undefined,
+  elevation: 0,
+  isSelected: false,
+};
+
+Card.ELEVATIONS = ELEVATIONS;
```
Comment (obedparla) [inline packages/react/src/components/Card/index.js]: Wouldn't it be cleaner to:

```
export {ELEVATIONS, Card}
export default Card 
```

Like this it makes me think I should/can do

```
import Card from ....
...
<>
   <Card elevation={Card.ELEVATIONS} />
<>
```

Which I'm not sure it's valid and can be confusing

File: packages/react/src/components/Card/index.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import PropTypes from "prop-types";
+import cx from "classnames";
+
+const ELEVATIONS = [0, 1, 2];
+
+const Card = ({ className, elevation, isSelected, ...props }) => (
+  <div
+    className={cx("Card", className, {
+      "is-selected": isSelected,
+      [`is-elevated--${elevation}`]: elevation,
```
Comment (pppemo) [inline packages/react/src/components/Card/index.js]:
```suggestion
      [`is-elevated--${elevation}`]: ELEVATIONS.includes(elevation),
```
?

File: packages/react/src/components/Card/index.js (hunk 2 above)
Comment (pppemo) [reply packages/react/src/components/Card/index.js]: I think @obedparla what you want is actually achieved by the current implementation, and the suggestion would disable it

File: packages/react/src/components/Card/index.js (hunk 3 above)
Comment (keanan) [reply packages/react/src/components/Card/index.js]: @pppemo won't that return true instead of the number?

Comment (keanan) [reply packages/react/src/components/Card/index.js]: ah sorry I see what's happening, you actually want it to return "true"

Comment (pppemo) [reply packages/react/src/components/Card/index.js]: Yeah, you want true. If it's a number, JS just casts it to boolean. But generally the value here is expected to be `boolean`.
And I thought that would be another guard (next to `propTypes`) that would cover a scenario, when user invokes `elevation={66609}` for example :)

Comment (keanan) [reply packages/react/src/components/Card/index.js]: Got you! Thanks for the insight.

Comment (ugogo) [reply packages/react/src/components/Card/index.js]: I was about to say that `0` is casts to `false`, but +1 for the extra scenario 🙂

File: packages/react/src/components/Card/index.js (hunk 2 above)
Comment (obedparla) [reply packages/react/src/components/Card/index.js]: You guys are right 😄

**Pull request merged**